### ISSUE TITLE
NVDA does not interact very well in IE ten if compatibility mode is turned off.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2876]
**Reported by Harich on 2012-12-18 15:41**
If the compatibility mode is turned off in Internet Explorer ten in Windows Eight, the NVDA does not work in some areas. For instance, the NVDA's own web site, nvda-project.org, in creating ticket, I could not interact with combo boxes. The NVDA would not turn focus mode.

**Comment by ehollig** [comment]
NVDA works well with IE 11 and I believe it works the same with IE 10 on Windows 8.1. Can you confirm this @bhavyashah?

**Comment by bhavyashah** [comment]
According to http://winaero.com/blog/how-to-enable-compatibility-view-in-internet-explorer-11-ie11/:
"In Internet Explorer 11, the compatibility view button is removed from the address bar because document modes are now deprecated. Microsoft felt that the button had outlived his purpose because it's been many years now since Microsoft introduced X-UA-Compatible tags. Web developers were supposed to add these X-UA-Compatible meta tags to their website's HTML header to indicate web page compatibility and not leave it to the user to depend on the button to correctly render the page. Now, Microsoft is expecting all web developers to update their websites to work with the latest edge mode and avoid using document modes, unless on a temporary basis. "
While I shall test the compatibility view in Internet Explorer 11 for the issue described (at a later time), I believe the above may aid in triaging this issue, and might even justify closing.

**Comment by ehollig** [comment]
Closing because of reasoning in https://github.com/nvaccess/nvda/issues/2876#issuecomment-319867001
